test/src/main/java/org/apache/accumulo/test/functional/FateStarvationIT.java

```diff
@@ -83,20 +85,26 @@ public void run() throws Exception {
 
       List<Text> splits = new ArrayList<>(TestIngest.getSplitPoints(0, 100000, 67));
 
-      List<Future<?>> futures = new ArrayList<>();
+      int numTasks = 100;
+      List<Future<?>> futures = new ArrayList<>(numTasks);
       var executor = Executors.newCachedThreadPool();
+      CountDownLatch startLatch = new CountDownLatch(32); // wait for a portion of the tasks to be
+                                                          // ready
```

Comment:
```suggestion
      // wait for a portion of the tasks to be ready
      CountDownLatch startLatch = new CountDownLatch(32);
```
Again here, is there a reason to not sync start of them all: `new CountDownLatch(numTasks)`